recipe/meta.yaml

```diff
@@ -1,8 +1,10 @@
 {% set name = "antlr4-python3-runtime" %}
+{% set old_name = "antlr-python-runtime" %}
 {% set version = "4.9.3" %}
 
+
 package:
-  name: antlr-python-runtime
+  name: {{ old_name|lower }}-meta
```

Comment:
maybe just be explicit everywhere... none of them are upper-cased anyway, and it gets pretty ugly down in the weeds with lots of template stuff